app/src/models/owner.ts

```diff
@@ -6,7 +8,8 @@ export class Owner {
   public constructor(
     public readonly login: string,
     public readonly endpoint: string,
-    public readonly id: number
+    public readonly id: number,
+    public readonly type?: GitHubAccountType
```

Comment:
Looks like we're not using the `hash` property on `Owner` any more which is great but we should either add this property to `.hash` or remove `.hash` entirely to stay consistent with the intent of `.hash`

Reply:
Decided to remove